manifest/process.go

```diff
@@ -37,6 +37,16 @@ func NewProcess(app string, s Service, m Manifest) Process {
 		}
 	}
 
+	for i, p := range s.Ports {
+		// Assign random ports to "internal" balancers
+		if !p.Public {
+			p.Public = true
```

Comment:
Yes #573 has the description for this problem. It was closed previously, but I reopened it because the problems still wasn't solved.

The goal here is to make `convox start` work when you have more than one internal load balancer listening on the same port.

This isn't a problem in production because each process type gets its own ELB.

It was broken with `convox start` because when we did the port mapping we'd just use a host port that matched the internal port. This caused port conflicts on the host if you had more than one process using the same internal port.

The solution here (as I understand it) was to use random host ports with `convox start` to avoid conflicts.